src/release/main.d

```diff
@@ -335,6 +332,28 @@ ActionList prepareMinorRelease ( ref HTTPConnection con, ref Repository repo,
 
 /*******************************************************************************
 
+    Prepares the actions to clar the release notes
+
+    Returns:
+        actions required to clear the release notes
+
+*******************************************************************************/
+
+ActionList clearReleaseNotes ( )
```

Comment:
I wonder if it would be better to clean release notes right _before_ tagging release. Would make merging of release notes more correct. CC @gavin-norman-sociomantic